BambooTracker.1

```diff
@@ -0,0 +1,25 @@
+.TH BambooTracker 1 "December 15 2018"
+.SH NAME
+BambooTracker \- YM2608 (OPNA) tracker
+.SH SYNOPSIS
+.B BambooTracker
+.RI " files" ...
+.br
+.SH DESCRIPTION
+This manual page documents briefly the
```

Comment:
```suggestion
This manual page briefly documents the
```
or
```suggestion
This manual page quickly summarizes the
```
would be better / more correct.

Reply:
I will fix it when I edit other documents.

Reply:
fixed 7a934324d9e569d2e62200cad954dda2c148f4fd